File: MyFrgment/app/src/main/java/com/example/myfrgment/MainActivity.java
package com.example.myfrgment;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button LlmarFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        LlmarFragment=(Button) findViewById(R.id.btnLlamar);
        LlmarFragment.setOnClickListener(this::onClick);

        //getSupportFragmentManager().beginTransaction().add(R.id.container,new FirstFragment()).commit();
    }
    public void onClick(View view){
        if (view.getId()==R.id.btnLlamar){
            getSupportFragmentManager().beginTransaction().replace(R.id.container,new FirstFragment()).commit();
            LlmarFragment.setVisibility(View.GONE);
        }

    }
}

File: HolaMundo/app/src/main/java/com/example/holamundo/MainActivity.kt
package com.example.holamundo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val n1=findViewById<EditText>(R.id.txt_nombre)
        val res=findViewById<TextView>(R.id.textR)
        val btn_sal=findViewById<Button>(R.id.btn_saludar)

        btn_sal.setOnClickListener {
            res.setText("HOLA "+(n1.text.toString()))
        }
    }
}